code/modules/mob/living/carbon/carbon.dm

```diff
@@ -248,19 +248,31 @@
 						)
 			// BEGIN HUGCODE - N3X
 			else
-				playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
 				if(M.zone_selected == "mouth")
-					M.visible_message(\
-					"<span class='notice'>[M] boops [src]'s nose.</span>",\
-					"<span class='notice'>You boop [src] on the nose.</span>",\
-					)
+					playsound(get_turf(src), 'sound/items/toysqueak1.ogg', 50, 1, -1)
+					if(isvox(M))
+						M.visible_message(\
+						"<span class='notice'>[M] boops [src]'s beak.</span>",\
+						"<span class='notice'>You boop [src] on the beak.</span>",\
+						)
+					else if(ismachineperson(M))
+						M.visible_message(\
+						"<span class='notice'>[M] boops [src]'s screen.</span>",\
+						"<span class='notice'>You boop [src] on the screen.</span>",\
+						)
+					else
+						M.visible_message(\
+						"<span class='notice'>[M] boops [src]'s nose.</span>",\
+						"<span class='notice'>You boop [src] on the nose.</span>",\
+						)
```

Comment:
Not a fan of such a mass of lines for just nose boops, but I dont really know any other way around it

Reply:
I do but it basically counts as obfuscation at that point...

Reply:
Just do something like this:
```dm
var/nose_type = isvox(M) ? "beak" : ismachineperson(M) ? "screen" : "nose"
M.visible_message(\
"<span class='notice'>[M] boops [src]'s [nose_type].</span>",\
"<span class='notice'>You boop [src] on the [nose_type].</span>",\
)
```

Reply:
I like it! That's almost what I was thinking of but way prettier :3